DDS-1294 [Tooltip] Retouching (#98)
dsv-rp/DDS: .changeset/afraid-eggs-kneel.md

---
"@daikin-oss/design-system-web-components": minor
---

> **BREAKING CHANGE** Tooltip: The `variant` attribute has been renamed to `color`. And the values `light` and `dark` have been renamed to `default` and `inverse` respectively.

Reviewer: It's not just a simple rename, because the meaning has changed.

```suggestion
**BREAKING CHANGE** Tooltip: The `variant` attribute has been replaced by the `color` attribute, which accepts `default` and `inverse` instead of `light` and `dark`.
```
